devel/external/AEGIS/README.md

```diff
@@ -0,0 +1,54 @@
+This README file describes the python code used to process AEGIS HST images in V and I bands to produce multi-band postage stamp images of galaxies. The selection procedure is explained in a separate [document](https://www.overleaf.com/read/krbzsccfdbpm). 
```

Comment:
This is also the document in external/AEGIS/document, right?  Which one is going to be the definitive version if they get out of sync?  

If it's the Overleaf version, then I think maybe we don't necessarily want it in the GalSim repo as well.  

If it's the GalSim version, then I'd suggest rewording this sentence to mention this version first.  e.g.
"... in a separate document in devel/external/AEGIS/document/main.tex; a rendered version of the document may be viewed at https://www.overleaf.com/read/krbzsccfdbpm"